### ISSUE TITLE
As Pho, I want to import and view all power generation devices for my group

### ISSUE THREAD

**Issue by dottorer** [buzzn/scrum#37]
PR: 

---
**Story:**
As PhO, I want to import and view all power generation devices for my group, so know what machinery is operated at that site.

---
**Definition of done**
- [ ] For any group, up to 4 power generation devices are shown.
- [ ] Each power generation device is correctly linked to the register with the right label.
- [ ] Each power generation device has the correct power source associated (sun, heat, water, wind).
- [ ] Each power generation device has manufacturer, technology, product_name, primary_energy, watt_peak, commissioning_date and law.  

**excluded from this sprint:** 

---

**Technical details**
- [ ]

**Comment by dottorer** [comment]
@evaklopp This will require a wireframe / UX design before we can size it. It is part of completing the group data load...

**Comment by evaklopp** [comment]
@dottorer does any of this devices needs an own detail site? How much information and what exactly is it? (Is it the same as we discussed the registers yesterday with PhO ?)

**Comment by dottorer** [comment]
@evaklopp Yes, this is exactly the same ticket. We should list all the power generation devices as a separate list with details. And we must link the register to one (or if needed more) of these devices.

**Comment by evaklopp** [comment]
@dottorer where can I find the details of a device? Had no luck in the console...

**Comment by dottorer** [comment]
@evaklopp It is not in the console, but here is what we discussed in the meeting yesterday from the beekeeper database:

![image](https://user-images.githubusercontent.com/14217285/33426257-7ed846d2-d5c1-11e7-8e8c-93c8b4163fa5.png)v

**Comment by evaklopp** [comment]
@dottorer thanks for the information. Why are we restricting the devices to 4? Can´t there be more?  Are there devices like batteries or ecars included aswell? is is only producing devices or can it be storing or consuming devices aswell?

**Comment by evaklopp** [comment]
@dottorer I have even more questions: so the details to show according to a device is not more than the categories we defined with PhO, am I correct?

**Comment by dottorer** [comment]
As far as i can tell that is correct.

**Comment by dottorer** [comment]
@evaklopp for any given register we should not attach more than 2 devices. As device categories we have PV and BHKW (we should add Wasserkraft und Wind) for generation devices. I don’t think we should offer consumption devices such as electric vehicles right now. Same for batteries. Let’s keep these as future features!

**Comment by evaklopp** [comment]
@dottorer what kind of numbers are the Peakleistung? W? kW? Sth else? Thanks

**Comment by evaklopp** [comment]
#112 detail page of the devices
